js.js

```diff
@@ -1,21 +1,39 @@
-const books = [];
 const addBtn = document.getElementById('add-new-book');
+const bttnn = document.getElementById('addbuttom');
 const booksoutput = document.querySelector('#booksoutput');
 
 function outputbooklist() {
-  const booklist = JSON.parse(localStorage.getItem('books'));
+  const booklistoutp = JSON.parse(localStorage.getItem('books'));
   booksoutput.innerHTML = '';
-  booklist.forEach((book, index) => {
+  booklistoutp.forEach((book, index) => {
     booksoutput.innerHTML += `<div class = "addedbook"><p>${book.title}</p><p>${book.author}</p><button onclick='removefromlist(${index})'>Remove</button></div`;
   });
 }
 
-function removefromlist(index) {
-  const booklist = JSON.parse(localStorage.getItem('books'));
-  booklist.splice(index, 1);
-  books.splice(index, 1);
-  localStorage.setItem('books', JSON.stringify(booklist));
-  outputbooklist();
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+
+  addtolist() {
+    let booklist = [];
+    if (localStorage.getItem('books') === null) {
+      booklist = [];
+    } else {
+      booklist = JSON.parse(localStorage.getItem('books'));
+    }
+    booklist.push(this);
+    localStorage.setItem('books', JSON.stringify(booklist));
+    outputbooklist();
+  }
+
+  removefromlist(index) {
+    const bookss = JSON.parse(localStorage.getItem('books'));
+    bookss.splice(index, 1);
```

Comment:
- [ ] The localstorage functionality doesn't work as expected, it should still retain the same number of books after reloading the page. Kindly fix it.